src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -126,13 +126,15 @@ export function refreshTransaction(
         });
 
         if (isFailedTransaction(transactionRefreshed) && analyticsSectionName) {
+          const errorMetadata =
```

Comment:
😞 Ugh, my bad. At first I thought it would be helpful if we kept those error response examples in a central location for future reference - but then I realized that this error response model already exists in https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/profile/postVet360Address and clearly indicates metadata is an array. An oversight on my end! Sorry!